shared/js/media/video_player.js

```diff
@@ -36,9 +39,11 @@ function VideoPlayer(container) {
   var poster = newelt(container, 'img', 'videoPoster');
   var player = newelt(container, 'video', 'videoPlayer');
   var controls = newelt(container, 'div', 'videoPlayerControls');
-  var playbutton = newelt(controls, 'button', 'videoPlayerPlayButton');
+  var playbutton = newelt(controls, 'button', 'videoPlayerPlayButton',
+                    'playbackPlay');
   var footer = newelt(controls, 'div', 'videoPlayerFooter hidden');
-  var pausebutton = newelt(footer, 'button', 'videoPlayerPauseButton');
+  var pausebutton = newelt(footer, 'button', 'videoPlayerPauseButton',
+                     'playbackPause');
```

Comment:
Nit: please fix the indentation here and above so that the wrapped argument lines up with the space after the open paren.  There's the same issue below, and I'd appreciate it if you'd fix that indentation as well, even though that already existed in the code.